Tests/SharedKeysTests.cc

```diff
@@ -97,6 +97,8 @@ TEST_CASE("decode", "[SharedKeys]") {
 
 TEST_CASE("revertToCount", "[SharedKeys]") {
     Retained<SharedKeys> sk = new SharedKeys();
+    sk->disableCaching();
```

Comment:
Whereas we want to ensure diableCaching does not affect the following checks, the other aspect is ensure get(Dict::key &keyToFind) works as expected.